2020-07-28 10:42:50,078 DEBUG   MainThread:32667 [wandb_config.py:_load_defaults():154] no defaults not found in config-defaults.yaml
2020-07-28 10:42:50,444 DEBUG   MainThread:32667 [meta.py:_setup_code_git():49] probe for git information
2020-07-28 10:42:50,477 DEBUG   MainThread:32667 [run_manager.py:__init__():547] Initialized sync for skin-mixmatch/291xaacc
2020-07-28 10:42:50,483 INFO    MainThread:32667 [run_manager.py:wrap_existing_process():1149] wrapping existing process 32553
2020-07-28 10:42:50,483 WARNING MainThread:32667 [io_wrap.py:register():108] SIGWINCH handler was not None: <Handlers.SIG_DFL: 0>
2020-07-28 10:42:50,825 INFO    MainThread:32667 [run_manager.py:init_run():928] system metrics and metadata threads started
2020-07-28 10:42:50,826 INFO    MainThread:32667 [run_manager.py:init_run():967] upserting run before process can begin, waiting at most 10 seconds
2020-07-28 10:42:51,227 INFO    Thread-15 :32667 [run_manager.py:_upsert_run():1056] saving pip packages
2020-07-28 10:42:51,229 INFO    Thread-15 :32667 [run_manager.py:_upsert_run():1058] initializing streaming files api
2020-07-28 10:42:51,230 INFO    Thread-15 :32667 [run_manager.py:_upsert_run():1065] unblocking file change observer, beginning sync with W&B servers
2020-07-28 10:42:51,231 INFO    MainThread:32667 [run_manager.py:wrap_existing_process():1166] informing user process we are ready to proceed
2020-07-28 10:42:51,236 INFO    MainThread:32667 [run_manager.py:_sync_etc():1298] entering loop for messages from user process
2020-07-28 10:42:51,440 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/config.yaml
2020-07-28 10:42:51,823 INFO    Thread-4  :32667 [run_manager.py:_on_file_created():681] file/dir created: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/requirements.txt
2020-07-28 10:42:51,825 INFO    Thread-4  :32667 [run_manager.py:_on_file_created():681] file/dir created: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:42:51,826 INFO    Thread-4  :32667 [run_manager.py:_on_file_created():681] file/dir created: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-metadata.json
2020-07-28 10:42:56,440 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:42:56,441 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-history.jsonl
2020-07-28 10:42:56,445 INFO    Thread-4  :32667 [run_manager.py:_on_file_created():681] file/dir created: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-summary.json
2020-07-28 10:43:05,446 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:43:07,447 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:43:07,448 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-metadata.json
2020-07-28 10:43:09,448 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:43:11,450 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:43:14,452 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:43:15,437 INFO    MainThread:32667 [run_manager.py:_sync_etc():1355] process received interrupt signal, shutting down
2020-07-28 10:43:15,438 INFO    MainThread:32667 [run_manager.py:_sync_etc():1408] closing log streams and sending exitcode to W&B
2020-07-28 10:43:15,439 INFO    MainThread:32667 [run_manager.py:shutdown():1072] shutting down system stats and metadata service
2020-07-28 10:43:15,452 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/output.log
2020-07-28 10:43:15,852 INFO    MainThread:32667 [run_manager.py:shutdown():1086] stopping streaming files and file change observer
2020-07-28 10:43:16,453 INFO    Thread-4  :32667 [run_manager.py:_on_file_modified():692] file/dir modified: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-metadata.json
2020-07-28 10:43:16,453 INFO    MainThread:32667 [run_manager.py:_on_file_created():681] file/dir created: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-events.jsonl
2020-07-28 10:43:18,945 INFO    MainThread:32667 [run_manager.py:_sync_etc():1420] process only ran for 29 seconds, not syncing files
48-291xaacc/wandb-metadata.json
2020-07-28 10:43:16,453 INFO    MainThread:32667 [291xaacc:run_manager.py:_on_file_created():681] file/dir created: /media/luckie/vol4/semi_supervised_cls/code/MixMatch-pytorch-master/wandb/run-20200728_104248-291xaacc/wandb-events.jsonl
2020-07-28 10:43:18,945 INFO    MainThread:32667 [291xaacc:run_manager.py:_sync_etc():1420] process only ran for 29 seconds, not syncing files
